### ISSUE TITLE
"Volltext" to "fulltext" issue rises again...

### ISSUE THREAD

**Issue by darupp** [hbz/laser-ygor#152]
... in the refactored version. Has already for the pre-refactored version fixed in #107.
https://app.zenhub.com/workspaces/gokb-5cb838424005bf783cb0ecd3/issues/hbz/laser-ygor/107

```
    "coverage" : [ {
      "coverageDepth" : "Volltext",
     ...
    } ],
```

**Comment by VolkerHBZ** [comment]
The the fix was implemented in a Class that got replaced by the refactoring. I need to find another solution...

**Comment by VolkerHBZ** [comment]
fixed in Ver 0.80 and up
